data/shaders/planet.frag

```diff
@@ -185,6 +185,29 @@ vec3 linearToSRGB(vec3 lin)
 
 void main()
 {
+#ifndef IS_MOON
+	if(sunInfo.w==0)
+	{
+		// We are drawing the Sun
+		vec4 texColor = texture2D(tex, texc);
+		texColor.rgb = srgbToLinear(texColor.rgb * sunInfo.rgb);
+		// Reference: Arthur N. Cox, "Allen's Astrophysical Quantities", 4th edition,

```

Comment:
please add year and publisher/location here or even better, add a full bib entry to the User Guide, with citation at the end of Appendix H.4.

Reply:
If I add a bib entry, how do I refer to it here?

Reply:
As for the year... I have no idea what year to use, and the publisher doesn't make it easier to determine it. The copyright year? The publication year of the given format? What?

I can just add DOI or ISBN, will it be sufficient?

Reply:
A short notification about visualization in the User Guide (see chapter for Solar system data) with citation of source ;) For the code the current info may be sufficient, but adding DOI or ISBN is very good addition.

Reply:
No, the .bib item is referred to in the User Guide where we refer to the sources of our collected wisdom. Here the reference should at least be complete enough with year. Right now it's good.

Reply:
Well, I have no idea where in the guide this citation should go, and whether it has to be hotlinked somewhere. I haven't found the "Solar system data" chapter or anything similar, so this doesn't help.
Since the citation here in the code is OK, I'm merging this PR.